ansible/memcached-operator/README.md

```diff
@@ -1,46 +1,105 @@
+$ cd operator-sdk-samples/ansible/memcached-operator
+```
 
-## Uninstall
+### Building the operator
 
-To uninstall the Deployment and the Memcached Ansible Operator, run the following commands
+Build the Memcached operator image and push it to a public registry, such as quay.io:
 
-1. Uninstall
-    ```bash
-    kubectl delete -f deploy/crds/cache_v1alpha1_memcached_cr.yaml
-    ```
-1. Uninstall Memcached Ansible Operator
-    ```bash
-    kubectl delete -f deploy/operator.yaml
-    ```
+```
+$ export IMAGE=quay.io/example-inc/memcached-operator:v0.0.1
+$ operator-sdk build $IMAGE
+$ docker push $IMAGE
+```
 
-Verify that the all pods created with the deployment are being `terminated` and are deleted
+**NOTE** The `quay.io/example-inc/memcached-operator` is an example. You should build and push the image for your repository. 
+
+### Using the image
+
+```
+# Update the operator manifest to use the built image name (if you are performing these steps on OSX, see note below)
+$ sed -i 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+# On OSX use:
+$ sed -i "" 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+```
+
+### Installing
+
+Run `make install` to install the operator. Check that the operator is running in the cluster, also check that the example Memcached service was deployed.
+
+Following the expected result. 
+
+```shell
+$ kubectl get all -n memcached
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/memcached-operator-69665475bf-zzf8j   2/2     Running   0          44s
+
+NAME                                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/memcached-operator-metrics   ClusterIP   10.109.140.17   <none>        8383/TCP   5s
+
+NAME                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/memcached-operator   1/1     1            1           44s
+
+NAME                                            DESIRED   CURRENT   READY   AGE
+replicaset.apps/memcached-operator-69665475bf   1         1         1       44s
+```
+
+### Uninstalling 
+
+To uninstall all that was performed in the above step run `make uninstall`.
+
+### Troubleshooting
+
+Use the following command to check the operator logs. 
+
+```
+kubectl logs deployment.apps/memcached-operator -c ansible
+```
+
+Use the following command to check the Ansible logs. 
+
+```
+kubectl logs deployment.apps/memcached-operator -c operator
+```
+
+**NOTE** To have further information about how to develop Ansible operators with [Operator-SDK][operator-sdk] check the [Ansible User Guide for Operator-SDK][ansible-guide]
+
+### Testing the Operator
+
+See [Testing Ansible Operators with Molecule][ansible-test-guide] documentation to know how to use the operator framework features to test it.  
+
+[operator-sdk]:  
+[dep_tool]: https://golang.github.io/dep/docs/installation.html
+[go_tool]: https://golang.org/dl/
```

Comment:
Same as above, not relevant to Ansible-based operators.

Reply:
👍